docker/README.md

```diff
@@ -13,16 +13,22 @@ in the project root.
 To start a RisingWave playground, run
 
 ```
+# Ensure risingwave image is of latest version
+docker pull ghcr.io/singularity-data/risingwave:latest
+# Start playground
 docker run -it ghcr.io/singularity-data/risingwave:latest
 ```
 
 To start a RisingWave cluster, run
 
 ```
+# Ensure risingwave image is of latest version
+docker pull ghcr.io/singularity-data/risingwave:latest
```

Comment:
Actually docker-compose will automatically pull the image specified in the yml.

Reply:
Well I didn't notice this pr enables auto merge. But anyway, I will prefer to strip off this line of command.

Reply:
docker-compose will pull this image only *when it doesn't exist*. However, it is possible that users have launched RisingWave months before, and want to try it again. During those months, the docker images could have been updated. At this time, docker-compose detects `latest` tag exists in local cache, and it will use the cached version months before.

Therefore, we should either specify a version tag here, or let users pull the image. Otherwise, they might be running an old version.